onnxscript/rewriter/no_op_test.py

```diff
@@ -177,6 +177,23 @@ def test_div_one_should_become_no_op_with_initializer(
         """
         )
 
+    @parameterized.parameterized.expand(
+        [
+            ("dropout zero", "ratio=0"),
+            ("dropout inference", "inference=1"),
+        ]
+    )
+    def test_dropout_zero_or_inference_no_op_with_initializer(self, _, attribute: str):
```

Comment:
Why doesn't it work

Reply:
Which one? Also: is the attribute `training_mode` or `inference`?

Reply:
Thanks! Corrected the attribute